### DISCUSSION TITLE
What Prometheus metrics would you like to see?

### DISCUSSION THREAD

**Discussion by ChappIO** [directus/directus#17457] [Archive]
In a project where I aim to use Directus, we also use Prometheus to collect metrics over time.

To that end, I created a metrics exporter extension for Directus: https://github.com/ChappIO/directus-extension-prometheus

Since I could not find one already, I figured I would open-source it. So, are there any metrics you would like to see added to this extension?

(If this is the wrong place for gathering feedback, I apologise. Let me know and I will delete this post.)

**Comment by rijkvanzanten** [comment]
Sweet! 

> (If this is the wrong place for gathering feedback, I apologise. Let me know and I will delete this post.)

It's not! I've pinned it to the top of discussions to get some more eyes on it 🙂

**Reply by ChappIO** [reply]
Sweet, I've also posted it to the discord. Let's see if people are even interested in this type of thing.

**Comment by kosssi** [comment]
It's a good idea.

I would like know number of users, files. The total size of files (maybe it's not possible).

Edit: I see `directus_collection_size{collection="directus_users"} 1` and `directus_collection_size{collection="directus_files"} 0` :+1:

**Reply by Clex1o1** [reply]
Allright lets see. 
Is it possible to hook into crud operations ? e.g. in hooks? For each collrection a count of entries, read/write operation, active user sessions, login errors, sign up errors. For realtime: active sockets. Thats my short brainstorm :D
